docs/pages/guides/using-sentry.mdx

```diff
@@ -162,17 +162,18 @@ Add `expo.hooks` property to your project's **app.json** or **app.config.js** fi
 }
 ```
 
-The `authToken` value can be generated from the [Sentry API page](https://sentry.io/settings/account/api/).
+To upload the source map to Sentry, you must create a [Sentry auth token](https://docs.sentry.io/product/cli/configuration/). After creating your Sentry auth token [here](https://sentry.io/settings/account/api/), you can configure this token through the `SENTRY_AUTH_TOKEN` environment variable in [EAS Build](/build-reference/variables/#using-secrets-in-environment-variables).
 
-<Collapsible summary="Prefer to use environment variables instead of storing values in app.json?">
+<Callout>
```

Comment:
```suggestion
<Callout type="warning">
```

Reply:
If you want to use the type, we have implemented similar mechanism to GH, so it will be:
```
> **warning** CONTENT
```